frontend/app/src/utils/model/index.ts

```diff
@@ -1,5 +1,48 @@
 import { kebabCase } from 'lodash-es';
-import type { Ref, WritableComputedRef } from 'vue';
+import type BigNumber from 'bignumber.js';
+import type { ModelRef, Ref, WritableComputedRef } from 'vue';
+
+export function useModelValueChild<
```

Comment:
Doesn't this look similar to https://github.com/rotki/rotki/blob/4d7ae90375c592a0cc4afec1fe66dce66d7961c9/frontend/app/src/utils/model/index.ts#L61